<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Background_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>941</width>
     <height>681</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>GUI Assets/login form.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="Email_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; color:#4ac8eb;&quot;&gt;Registered Email:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Password_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; color:#4ac8eb;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Email_input">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="Password_input">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="Login_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semibold</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Heading_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>821</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600; color:#4ac8eb;&quot;&gt;Login for existing users&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Register_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>611</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#4ac8eb;&quot;&gt;Don't have an account? Please click on register button:-&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Register_button">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>490</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semibold</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Reset_password_button">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>270</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset passoword</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Hidden_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>320</y>
     <width>381</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Hidden Text&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="internet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>871</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:600; color:#ff0000;&quot;&gt;Please connect to internet&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>Reset_password_button</zorder>
  <zorder>Background_label</zorder>
  <zorder>Email_label</zorder>
  <zorder>Password_label</zorder>
  <zorder>Email_input</zorder>
  <zorder>Password_input</zorder>
  <zorder>Login_button</zorder>
  <zorder>Heading_label</zorder>
  <zorder>Register_label</zorder>
  <zorder>Register_button</zorder>
  <zorder>Hidden_label</zorder>
  <zorder>internet</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
